src/test/regress/sql/multi_function_evaluation.sql

```diff
@@ -0,0 +1,118 @@
+--
+-- MULTI_FUNCTION_EVALUATION
+--
+
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1190000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1190000;
+
+-- fields with SERIAL work
+
+CREATE TABLE example (key INT, value SERIAL);
+SELECT master_create_distributed_table('example', 'key', 'hash');
```

Comment:
Unfortunately there's no way to avoid connecting to the workers. If you use an INT type and then SET DEFAULT that also gets propagated to the workers.